src/types/emotion.d.ts

```diff
@@ -0,0 +1,19 @@
+import '@emotion/react';
+import { SerializedStyles } from '@emotion/react';
+
+declare module '@emotion/react' {
+  export interface Theme {
+    color: {
+      primary: string;
+      positive: string;
+      negative: string;
+    };
+  }
+}
+
+// Prevents errors with using 'CSS' prop on React components
+declare module 'react' {
```

Comment:
Double-check if this is needed! I'd hope not now :)

Reply:
I tried removing it, and just got a question.

If I also remove the following from the top of this file, _that's_ when I seem to get an error now, but the react module removal was okay.
`import '@emotion/react';
import { SerializedStyles } from '@emotion/react';`
![image](https://user-images.githubusercontent.com/9061026/208829718-aae8c97f-f62f-4b72-9434-f27d26040eb2.png)

https://emotion.sh/docs/typescript#define-a-theme
It does that here, too, so I'll leave that import in since that's necessary for the Theme part.